modules/platforms/cpp/core-test/src/cache_test.cpp

```diff
@@ -166,6 +166,8 @@ struct CacheNativePersistenceTestSuiteFixture
         grid0 = Ignite();
 
         Ignition::StopAll(true);
+
+        ignite_test::ClearLfs();
```

Comment:
We should also do it in constructor, just in case someone forgot to clean LFS previously.